Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -327,6 +327,28 @@ theorem min_radius_le_radius_add (p q : FormalMultilinearSeries 𝕜 E F) :
 theorem radius_neg (p : FormalMultilinearSeries 𝕜 E F) : (-p).radius = p.radius := by
   simp only [radius, neg_apply, norm_neg]
 
+theorem radius_le_smul {p : FormalMultilinearSeries 𝕜 E F} {c : 𝕜} : p.radius ≤ (c • p).radius := by
+  simp only [radius, smul_apply]
+  apply iSup_mono
+  intro r
+  apply iSup_mono'
+  intro C
+  use ‖c‖ * C
+  apply iSup_mono'
+  intro h
+  simp
+  intro n
+  rw [norm_smul c (p n), mul_assoc]
+  apply mul_le_mul_of_nonneg_left
+  · apply h
+  · simp
```

Comment:
Here I just put all the `iSup_mono` applications on a single line (`refine` is nice for this sort of thing), I squeezed the non-terminal `simp`, and used `gcongr` instead of calling `mul_le_mul_of_nonneg_left` directly.
```suggestion
  simp only [radius, smul_apply]
  refine iSup_mono fun r ↦ iSup_mono' fun C ↦ ⟨‖c‖ * C, iSup_mono' fun h ↦ ?_⟩
  simp only [le_refl, exists_prop, and_true]
  intro n
  rw [norm_smul c (p n), mul_assoc]
  gcongr
  exact h n
```